new/PRS13277Abbelo.xml

```diff
@@ -0,0 +1,71 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾAbbǝlo</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="grc">Greek</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2020-12-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አብሎ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAbbǝlo</persName>
+                    <persName xml:lang="gez" xml:id="n2">አበሉ፡</persName>
+                    <persName xml:lang="gez" corresp="#n2">ʾAbbalu</persName>
+                    <persName xml:lang="grc" xml:id="n3">Apollo</persName>
+                    <faith type="Christianity"/>
+                    <occupation type="ecclesiastic"/>
+                    <nationality type="Egypt"/>
+                    <birth/>
+                    <death/>
+                    <floruit>According to a narrative in <ref type="mss" corresp="EMML1840">he was a monastic figure
+                    contemporary to <persName ref="PRS2209Athanasi">Athanasius of Alexandria</persName> according to
+                    a narrative in <ref type="mss" corresp="EMML1840">EMML 1840</ref>. In the same text he is referred to as
+                  "resembling angels" (<foreign xml:lang="gez">ዘይመስል፡ መላእክት፡</foreign>) which might suggest an identification
+                with Apollo mentioned in the Coptic Synaxary on the 25th of Bābah.</ref></floruit>
+                </person>
```

Comment:
Here you have the expression "according to" twice, at the beginning and at the end of a sentence. Is there a particular ms_Item, in which he is mentioned?

Reply:
Yes, but the manuscript is not yet catalogued.